src/database/index.ts

```diff
@@ -567,6 +578,17 @@ export class Database {
     return application ?? null;
   }
 
+  async getApplicationsByProjectId(chainId: ChainId, projectId: Hex) {
+    const applications = await this.#db
+      .selectFrom("applications")
+      .where("chainId", "=", chainId)
+      .where("projectId", "=", projectId)
+      .selectAll()
+      .execute();
+
+    return applications ?? null;
```

Comment:
```suggestion
    return applications;
```

this will never be null right? it's always an array